DOCS/man/options.rst

```diff
@@ -6267,6 +6267,39 @@ them.
 
     macOS and cocoa-cb only.
 
+``--cocoa-cb-output-csp=<csp>``
+    This sets the color space of the layer to activate the macOS color
+    transformation. Depending on the color space used the system's EDR (HDR)
+    support will be activated. To get correct results, this needs to be set to
+    the color primaries/transfer characteristics of the video. It is recommended
```

Comment:
> of the video

I think it is more correct to say that it has to have the same characteristics as VO target.

Reply:
indeed true.